site/smart-accounts/session-keys/getting-started.md

```diff
@@ -0,0 +1,183 @@
+      spendLimit: 1000000n,
+    });
+
+  const { hash } = await extendedClient.installSessionKeyPlugin({
+    // 1st arg is the initial set of session keys
+    // 2nd arg is the tags for the session keys
+    // 3rd arg is the initial set of permissions
+    args: [
+      [await sessionKeySigner.getAddress()],
+      ["0x0"],
+      [initialPermissions.encode()],
+    ],
+  });
+
+  await extendedClient.waitForUserOperationTransaction({ hash });
+}
+```
+
+### Construct the initial set of permissions
+
+Session keys are powerful because of permissions that limit what actions they can take. When you add a session key, you should also specify the initial permissions that apply over the key.
+
+**Default Values**
+
+Permissions start with the following default values:
+
+| Permission               | Default Value                                                                                                                                    |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Access control list      | Type: allowlist <br /> The list starts empty. When the allowlist is empty, all calls will be denied.                                             |
+| Time range               | Unlimited                                                                                                                                        |
+| Native token spend limit | 0 <br /> This means all calls spending the native token will be denied, unless the limit is updated or removed.                                  |
+| ERC-20 spend limit       | Unset. If you want to enabled an ERC-20 spend limit, add the ERC-20 token contract to the access control list and set the spending limit amount. |
+| Gas spend limits         | Unset. When defining the session key’s permissions, you should specify either a spending limit or a required paymaster.                          |
+
+Importance of gas limits
+
+Gas spend limits are critically important to protecting the account. If you are using a session key, you should configure either a required paymaster rule or a gas spend limit. Failing to do so could allow a malicious session key to drain the account’s native token balance.
+
+::: details View the full set of supported permissions here
+
+<!-- TODO move this in to the docs section in packages overview --->
+
+<<< @/../packages/accounts/src/msca/plugins/session-key/permissions.ts
+:::
+
+Let's use the permission builder to build a set of permissions that sets a spend limit:
+
+```ts
+const initialPermissions =
+  new SessionKeyPermissionsBuilder().setNativeTokenSpendLimit({
+    spendLimit: 1000000n,
+  });
+
+const result = await extendedClient.updateKeyPermissions({
+  args: [sessionKeyAddress, initialPermissions.encode()],
+});
+```
+
+## Creating, deleting, and editing session keys
```

Comment:
I'll follow up with a PR that makes the API for these methods similar to the `remove` method so that the APIs are consistent

Reply:
done in https://github.com/alchemyplatform/aa-sdk/pull/430